### PR TITLE
speedtest-cli: add --csv and --json examples

### PR DESCRIPTION
**Pull request by jpiamjariyakul**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

**Review by navarroaxel — APPROVED** [review]

File: pages/common/speedtest-cli.md [hunk 1]
```diff
@@ -18,3 +22,11 @@
 - Run a speed test to the given speedtest.net server id:
 
 `speedtest-cli --server {{server_id}}`
+
+- Generate CSV results file with specified delimiter:
+
+`speedtest-cli --csv --csv-delimiter {{delimiter_char}}`
+
+- Suppress verbose output & display results in JSON format:
```
Comment (navarroaxel) [inline pages/common/speedtest-cli.md:30]:
```suggestion
- Display results in JSON format, suppressing the progress information:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by jpiamjariyakul — COMMENTED** [review]
Merged writeup with @navarroaxel's edits

**Review by owenvoke — APPROVED** [review]

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Just 1 additional comment.

File: pages/common/speedtest-cli.md [hunk 2]
```diff
@@ -18,3 +22,11 @@
 - Run a speed test to the given speedtest.net server id:
 
 `speedtest-cli --server {{server_id}}`
+
+- Generate CSV results file with specified delimiter:
```
[the remark this answers is not in the document]
Comment (jpiamjariyakul) [reply pages/common/speedtest-cli.md:26]: Thanks for the correction!

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**